src/main/java/ch/admin/bag/covidcertificate/service/CovidCertificatePdfGenerateRequestDtoMapperService.java

```diff
@@ -65,7 +65,7 @@ public VaccinationTouristCertificatePdf toVaccinationTouristCertificatePdf(Vacci
     }
 
     public TestCertificatePdf toTestCertificatePdf(TestCertificatePdfGenerateRequestDto pdfGenerateRequestDto) {
-        var testValueSet = valueSetsService.getIssuableTestDto(pdfGenerateRequestDto.getDecodedCert().getTestInfo().get(0).getTypeOfTest(), pdfGenerateRequestDto.getDecodedCert().getTestInfo().get(0).getTestManufacturer());
+        var testValueSet = valueSetsService.validateAndGetIssuableTestDto(pdfGenerateRequestDto.getDecodedCert().getTestInfo().get(0).getTypeOfTest(), pdfGenerateRequestDto.getDecodedCert().getTestInfo().get(0).getTestManufacturer());
```

Comment:
Repeatedly using "pdfGenerateRequestDto.getDecodedCert().getTestInfo().get(0)" should be replaced by extraction to value and referencing, imho